lib/cli/src/commands/login.rs

```diff
@@ -1,86 +1,122 @@
+use std::path::PathBuf;
+
 use clap::Parser;
 #[cfg(not(test))]
 use dialoguer::Input;
-use wasmer_registry::WasmerConfig;
+
+use crate::{Registry, WasmerEnv};
 
 /// Subcommand for listing packages
 #[derive(Debug, Clone, Parser)]
 pub struct Login {
-    /// Registry to log into (default: wasmer.io)
-    #[clap(long, default_value = "wasmer.io")]
-    pub registry: String,
-    /// Login token
-    #[clap(name = "TOKEN")]
+    /// Set Wasmer's home directory
```

Comment:
Why does this not also use `WasmerEnv`?

I get that the particular can be different, but that should be unifiable with some focused methods on WasmerEnv?